### PR TITLE
Gem jar divya api

### PR DESCRIPTION
**Pull request by gem-divyamadan**
@Gem-NeerajSingh kindly review the Dashboardprofile module.

### CODE REVIEW

File: src/main/java/com/qa/mis/stepdefinitions/DashboardProfileSteps.java [hunk 1]
```diff
@@ -16,7 +16,7 @@ public void LoginDshbrdAuth(String url, String method) throws Exception {
         status = Utils.LoginUser(url, method, token, "LoginUser").getStatus();
     }
 
-    @Then("Verify status code {int}")
+    @Then("Verify dashboard status code {int}")
     public void check_status_code(int Expected) {
         Utils.VerifyStatusCode(Expected, status);
     }
```
Comment (Gem-NeerajSingh) [inline src/main/java/com/qa/mis/stepdefinitions/DashboardProfileSteps.java:22]: try catch or throws missing

File: src/main/resources/features/DashboardProfile.feature [hunk 2]
```diff
@@ -2,129 +2,129 @@ Feature: MIS Portal API Login Authentication
...
       | postApi  | post   | 200            |
 
   Scenario Outline: API VALIDATION: User Profile Data
     Given Set the endpoint "<endpoint>" , method "<method>" and "<sampleName>" for User Profile Data
-    Then Verify status code <expectedStatus>
+    Then Verify dashboard status code <expectedStatus>
     Examples:
 
       | endpoint        | method | expectedStatus | sampleName  |
-      | UserProfileData | post   | 200            | UsePrflData |
```
Comment (Gem-NeerajSingh) [inline src/main/resources/features/DashboardProfile.feature:16]: Replace sample name with payload

Comment (Gem-NeerajSingh) [reply src/main/resources/features/DashboardProfile.feature:16]: and apply that is all feature file and step definition

**Review by Gem-NeerajSingh — CHANGES_REQUESTED** [review]
minior changes req

File: src/main/java/com/qa/mis/stepdefinitions/DashboardProfileSteps.java [hunk 3]
```diff
@@ -15,7 +15,7 @@ public void LoginDshbrdAuth(String url, String method) throws Exception {
         token.put("Authorization", Utils.GetAuthorization());
         status = Utils.LoginUser(url, method, token, "LoginUser").getStatus();
     }
-
+    
     @Then("Verify dashboard status code {int}")
     public void check_status_code(int Expected) {
```
Comment (Gem-NeerajSingh) [inline src/main/java/com/qa/mis/stepdefinitions/DashboardProfileSteps.java:20]: exception handling missing

File: src/main/java/com/qa/mis/stepdefinitions/FormsSteps.java [hunk 4]
```diff
@@ -1,81 +1,85 @@
...
     int status;
+
     @Given("^Set endpoint \"(.*)\" and method \"(.*)\" for MIS login$")
     public void Login(String url, String method) throws Exception {
-        HashMap<String,String> token=new HashMap<String,String>();
+        HashMap<String, String> token = new HashMap<String, String>();
         token.put("Authorization", Utils.GetAuthorization());
-        status = Utils.LoginUser(url, method,token,"Successful Login").getStatus();
+        status = Utils.LoginUser(url, method, token, "").getStatus();
     }
```
Comment (Gem-NeerajSingh) [inline src/main/java/com/qa/mis/stepdefinitions/FormsSteps.java]: Any specific reason for the blank string

File: src/main/java/com/qa/mis/stepdefinitions/FormsSteps.java [hunk 5]
```diff
@@ -1,81 +1,85 @@
...
 
-    @Then("Verify status code {int}")
+    @Then("Verify Forms status code {int}")
     public void check_status_code(int Expected) {
         Utils.VerifyStatusCode(Expected, status);
```
Comment (Gem-NeerajSingh) [inline src/main/java/com/qa/mis/stepdefinitions/FormsSteps.java]: exception handling missing

**Comment by gem-divyamadan** [comment]
Done

**Pull request closed without merging**